### ISSUE TITLE
How to represent results when the id has 2 or more correspondences

### ISSUE THREAD

**Issue by nunogit** [bridgedb/Goliath#6]
Example:

HMDB01877

results in (look at chebi)

CHEBI:39867	ChEBI
3121	PubChem-compound
115217	ChEBI
HMDB0001877	HMDB
NIJJYAXOARWZEE-UHFFFAOYSA-N	InChIKey
39867	ChEBI
CHEBI:115217	ChEBI
Q240642	Wikidata
LMFA01020291	LIPID MAPS
3009	Chemspider
DTXSID6023733	EPA CompTox
C07185	KEGG Compound
HMDB01877	HMDB
99-66-1	CAS

Other Id's that can serve as example:

HMDB00112 | 16865 |  
HMDB00295 | 17659 |  
HMDB00901 | undefined |  
HMDB00935 | CHEBI:17200 |  
HMDB01275 | 15539 |  
HMDB01877 | CHEBI:39867 |  
HMDB13037 | 7981

**Comment by DeniseSl22** [comment]
This will happen for other databases as well.... Could we include a checkmark button, where people can select if they want one mapping, or all mappings? And then add the "all mappings" as an array (or as separate rows?!). And, how is the "one mapping" selected; just the first one that is encountered in the bridgedb webservice? How stable is that selection procedure?

**Comment by nunogit** [comment]
Part of this discussion is now on #11 (the part relating equivalent Id's. Eg.: chebi:12345  =  12345)

**Comment by DeniseSl22** [comment]
No, the unification of results is a different discussion in my opinion. This issue is concerning the one vs many mappings, and how to display those results on the website.

**Comment by nunogit** [comment]
@DeniseSl22 exactly, that's why I mentioned that part of this issue is now in #11. Initially I grouped both problems here.
